.github/workflows/test-compose-consumer.yml

```diff
@@ -0,0 +1,24 @@
+name: "Test shot-compose-consumer"
+on: [pull_request, push]
+env:
+  GRADLE_OPTS: "-Dorg.gradle.jvmargs=-Xmx4g -Dorg.gradle.daemon=false -Dkotlin.incremental=false"
+jobs:
+  test:
+    name: Run tests for shot-consumer-compose
+    runs-on: macos-latest
```

Comment:
Do you need to run it on iOS or could it run on Linux too?
Just saying as mac runner cost/consume 10x more than the Linux one.

Reply:
The Android emulator is faster in this distribution 😃